validation/src/main/scala/hmda/validation/rules/lar/macro/Q074.scala

```diff
@@ -0,0 +1,49 @@
+package hmda.validation.rules.lar.`macro`
+
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
+import com.typesafe.config.ConfigFactory
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.dsl.PredicateCommon._
+import hmda.validation.dsl.PredicateSyntax._
+import hmda.validation.dsl.Result
+import hmda.validation.rules.AggregateEditCheck
+import hmda.validation.rules.lar.`macro`.MacroEditTypes._
+
+import scala.concurrent.{ ExecutionContext, Future }
+
+object Q074 extends AggregateEditCheck[LoanApplicationRegisterSource, LoanApplicationRegister] {
+
+  val config = ConfigFactory.load()
+  val count = config.getInt("hmda.validation.macro.Q074.numOfLoanApplications")
+  val multiplier = config.getDouble("hmda.validation.macro.Q074.numOfLarsMultiplier")
+
+  override def name = "Q074"
+
+  override def apply(lars: LoanApplicationRegisterSource)(implicit system: ActorSystem, materializer: ActorMaterializer, ec: ExecutionContext): Future[Result] = {
+
+    val purchaserType =
+      count(lars.filter(lar =>
+        lar.loan.purpose == 3
+          && Seq(1, 6).contains(lar.actionTakenType)
+          && Seq(1, 2).contains(lar.loan.propertyType)
+          && Seq(2, 3).contains(lar.loan.loanType)))
+
+    val sold =
+      count(lars.filter(lar =>
+        lar.loan.purpose == 3
+          && lar.actionTakenType == 6
```

Comment:
See the comment [here](https://github.com/cfpb/hmda-platform/issues/240#issuecomment-263991392) for clarification on the logic of the edit. The count of loans sold will have to filter on the `purchaserType` field.